src/style.css

```
@@ -0,0 +1,40 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  height: 15px;
+}
+
+.stars__star {
+  width: 15px;
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars--0 :nth-child(n) {
+  background-image: url(images/star.svg);
+}
+
+.stars--1 :nth-child(n+2) {
+  background-image: url(images/star.svg);
+}
+
+.stars--2 :nth-child(n+3) {
+  background-image: url(images/star.svg);
+}
+
+.stars--3 :nth-child(n+4) {
+  background-image: url(images/star.svg);
+}
+
+.stars--4 :nth-child(n+5) {
+  background-image: url(images/star.svg);
+}
+
+.stars--5 :nth-child(n+6) {
+  background-image: url(images/star.svg);
+}
```

these style can be combined together like this
```suggestion
.stars--4 :nth-child(n+5),
.stars--5 :nth-child(n+6) {
  background-image: url(images/star.svg);
}
```

---

It is a good remark :wink: